### ISSUE TITLE
Pathplanner: zu viele Erokesen

### ISSUE THREAD

**Issue by marvin78** [EinEinfach/CaSSAndRA#143]
Ich weiß, dass du nicht mehr gerne am Pathplanner fuckelst aber das hier stört mich und dürfte, aus meiner Sicht, auch so nicht sein:

![image](https://github.com/EinEinfach/CaSSAndRA/assets/10920356/716889eb-857e-4ab0-814e-134bcb724648)

Da fehlt zu viel und es bleibt hier immer eine ganze Menge stehen: So müsste das aussehen (lieber zu viel, als zu wenig):

![image](https://github.com/EinEinfach/CaSSAndRA/assets/10920356/0ad08ef9-3e2d-438d-a17a-e5e9ddc4a417)

Wenn du Zeit hast, kannst du dir das ja ggf. nochmal ansehen. Ich habe das auf meiner Map in vielen Bereichen.

**Comment by max951** [comment]
+1
ist bei mir leider auch so

**Comment by EinEinfach** [comment]
Ja, muss ich mir genauer anschauen

**Comment by themanfrommoon** [comment]
+1
Me, too

Es stört ja nicht, wenn der Rover 2x über eine Stelle drüber fährt, aber keinmal ist einmal zu wenig.

**Comment by EinEinfach** [comment]
Das Theme ist echt knifflig, habe tatsächlich noch keine simple Lösung für das Problem.

Als Übergangslösung schlage ich euch vor distance to border = border laps im Pathplanner auszuwählen

**Comment by marvin78** [comment]
Du meinst bei disatcne to border=2 auch border laps=2. So ist es hier eingestellt. Man könnte 3 border Laps nehmen aber bei unserem komplexen Garten dauert das Mähen einer Fläche gut doppelt so lange. Distance to border zu verringern ergibt bei Alfred keinen Sinn. Dann fährt er gegen alles und jeden.

**Comment by EinEinfach** [comment]
Ah sorry,
ich meine border laps = distance to border + 1

**Comment by marvin78** [comment]
Ja. Das ist ein Problem. Die Idee hatte ich auch. Unser Garten ist jedoch zu komplex dafür. Die Border-Laps sind das, was ohnehin schon prozentual (im Verhältnis zur Fahrdauer) am längsten dauert und die meiste Energie aus dem Akku zieht. Noch eine Runde mehr pro Task ist hier nicht sinnvoll. Ich habe keine Ahnung davon, wie komplex das ist, das im Pathplanner zu lösen. Ich würde mir bloß wünschen, dass es nicht aus dem Fokus verschwindet. :)

Manchmal kann man es mit anderen Winkeln lösen. Aber auch das kommt nicht der Effizienzu zu Gute.

**Comment by EinEinfach** [comment]
Wie gesagt, erstmal als workaround gedacht. Ich habe das Problem identifiziert, aber habe noch keine 100%e Lösung, ein Teil der Löcher kann ich schon stopfen, aber bei weitem nicht alle. 

Es ist jetzt als issue dokumentiert, und geht nicht verloren

**Comment by marvin78** [comment]
Gut, Danke. Wir haben einen Garten mit vielen Ecken und Kanten und ich habe aktuell 20 Stellen, an denen große Irokesen stehen bleiben (und noch viele kleine Ecken). Wenn das Gras nicht so schnell wächst, wie aktuell, dann sieht man es nicht ganz so schlimm (und man kann es eben lösen, in dem man täglich den Winkel wechselt) aber aktuell sieht das echt besch*** aus, da das Gras hier mehrere cm am Tag wächst und es auch nicht überall sinnvoll ist, den Winkel zu wechseln. 

Allerdings muss ich auch sagen, dass Alfred aktuell mit leicht modifizierter 318 Firmware und vor allem danke Cassandra, einen echt guten Job macht. Von daher ist das alles jammern auf hohem Niveau. Das ist ggf. auch das Problem: Setzt man den Standard hoch, werden auch die Ansprüche immer höher ;)

**Comment by EinEinfach** [comment]
Ich kann schon mal das Update pushen, wo zumindest ein Teil der Irokesen verschwinden soll

**Comment by marvin78** [comment]
Kannst dir auch Zeit lassen. Ich kann das ggf. sowieso erst heute Abend testen. Alfred mäht gerade.

**Comment by marvin78** [comment]
Jetzt sieht das an der einen Stelle so aus:

![image](https://github.com/EinEinfach/CaSSAndRA/assets/10920356/660c9a12-e794-4e07-8fd7-8bb04f10feac)

Das darf da so nicht sein. Da fährt in die Ecke mit Mauer und Stufe. Insgesamt erscheint da viel mehr Durcheinander.

**Comment by EinEinfach** [comment]
Gleiche Einstellungen wie vorhin?

**Comment by marvin78** [comment]
Jap. Gleiche Einstellungen, wie vorhin. Interessant ist, dass es aber offenbar gar nicht an der neuen Version liegt. Ich hatte wieder zurück gerüstet auf den vorherigen Kommit und er macht den gleichen Quatsch (auch mit derm ganzen Durcheinander hin und her). Nur der Original Task (jap, gleiche Einstellungen - der existiert schon lange). sieht ordentlich aus (eben mit den Haken an den ein oder anderen Stellen).
